### ISSUE TITLE
[Bug]: Multiple authors from matched tab are set as a single author instead of separate authors

### ISSUE THREAD

**Issue by kymahi** [advplyr/audiobookshelf#667]
### Describe the issue

When matching a book, if the book has multiple authors (comma separated typically), the author field will be populated with [John Smith, Joe Schmo] instead of [John Smith], [Joe Schmo]

### Steps to reproduce the issue

1. Add a book with multiple authors to server
2. Edit newly added book
3. Navigate to Match tab
4. Search for book on any source
5. Find matched book and select it
6. Save changes, allowing overwriting
7. See that there is a single author entry with two, comma-separated names instead of two authors

### Audiobookshelf version

v2.0.18

### How are you running audiobookshelf?

Docker

**Comment by deranjer** [comment]
I believe this same issue is occurring with podcasts and the "Genres" field.  I was matching on "Dan Carlin's Hardcore History" podcast, it had genres "History,Podcasts" and that failed to update with this error:

```
ad6eb8f.js:2 TypeError: this.selectedMatch[l].split is not a function
    at f.buildMatchUpdatePayload (5c140c4.js:1:13494)
    at 5c140c4.js:1:14102
    at v (ad6eb8f.js:2:74429)
    at Generator._invoke (ad6eb8f.js:2:74217)
    at Generator.next (ad6eb8f.js:2:74858)
    at r (ad6eb8f.js:2:67421)
    at l (ad6eb8f.js:2:67624)
    at ad6eb8f.js:2:67683
    at new Promise (<anonymous>)
    at ad6eb8f.js:2:67564
re @ ad6eb8f.js:2
ne @ ad6eb8f.js:2
te @ ad6eb8f.js:2
(anonymous) @ ad6eb8f.js:2
Promise.catch (async)
ee @ ad6eb8f.js:2
n @ ad6eb8f.js:2
Dr.c._wrapper @ ad6eb8f.js:2
ad6eb8f.js:2 TypeError: e.book.genres.join is not a function
    at f.<anonymous> (5c140c4.js:1:7396)
    at t._render (ad6eb8f.js:2:36879)
    at f.r (ad6eb8f.js:2:66644)
    at jn.get (ad6eb8f.js:2:28187)
    at jn.run (ad6eb8f.js:2:29071)
    at En (ad6eb8f.js:2:27170)
    at Array.<anonymous> (ad6eb8f.js:2:13369)
    at se (ad6eb8f.js:2:12813)
```

Removing the "Podcasts" genre along with the comma allowed it to import.

**Comment by advplyr** [comment]
Fixed in [v2.0.19](https://github.com/advplyr/audiobookshelf/releases/tag/v2.0.19)
